sync_ci/Makefile

```diff
@@ -30,6 +30,7 @@ install:
 	@mkdir -p $(INSTALL_DIR)
 	@make binary
 	@mv bin/sync-ci-service $(INSTALL_DIR)/sync-ci-service
+	@cp sync_ci/parser/envrules.json $(INSTALL_DIR)/envrules.json
```

Comment:
specify the `rp` in the start command. such as 
`ExecStart=$(INSTALL_DIR)/sync-ci-service sync-ci -dsn "${dsn}" -lp "$(INSTALL_DIR)/log" -rp "$(INSTALL_DIR)/envrules.json"`